frontend/src/app/app.routes.ts

```diff
@@ -1,19 +1,26 @@
 import { Routes } from '@angular/router';
 
 import { HomeComponent } from './home/home.component';
+import { LoginComponent } from '~/app/login/login.component';
+import { AuthGuard } from '~/app/auth/auth.guard';
 
 export const routes: Routes = [
   {
-      path: '',
-      redirectTo: '/transactions',
-      pathMatch: 'full',
+    path: '',
+    redirectTo: '/home',
+    pathMatch: 'full',
```

Comment:
Could you move it to another module? For instance `Pages`.

Reply:
Done